Homework/Quang_Minh/HomeWorkLesson9/PowXN.java

```diff
@@ -0,0 +1,11 @@
+package HomeWorkLesson9;
+
+public class PowXN {
```

Comment:
althoght it works, i would not recommend doing this in an interview. hard to explain